crates/swc_css_utils/src/named-colors.json

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
Please modify `swc_css_utils/Cargo.toml` to include `src/**/*.json` in `includes`